migrate/migrate_repos.py

```diff
@@ -105,5 +231,128 @@ def extract_repo_names(repo_list_file):
     return empty_lines_removed
 
 
+def load_permissions(teams_file, api, dest_org, skip_missing_teams):
+    """
+    Return a dict mapping of team_slug to RepoPermissions.
+
+    These include the team permissions we're planning to apply. If
+    skip_missing_teams is True, we'll strip out all missing teams and users
+    from the returned RepoPermissions objects.
```

Comment:
Really thorough handling & reporting of missing teams and users here. Looks great.